src/globus/widgets/file_manager.ts

```diff
@@ -50,18 +77,22 @@ export class GlobusFileManager extends Widget {
 
         this.title.label = 'File Manager';
 
+        this.update();
+    }
+
+    onUpdateRequest() {
+        removeChildren(this.node);
         this.createHTMLElements();
     }
 
```

Comment:
Add a comment bit of what this does, and what errors it can raise. Make sure that whoever calls this handles those exceptions.